packages/react/src/lib/positioner/Popper.tsx

```diff
@@ -43,6 +45,24 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
 
   React.useEffect(
     () => {
+      const pointerTargetRefElement = pointerTargetRef && pointerTargetRef.current
+      const popperHasScrollableParent = getScrollParent(contentRef.current) !== document.body
+
+      const modifiers: PopperJS.Modifiers = _.merge(
+        popperHasScrollableParent && {
```

Comment:
may we leave comments for each of the modifiers used here - like short explanation that suggests the case it is aimed to address? See it necessary for the first and the last one in the list

Reply:
done, thanks